dbf2csv/wizard/dbf2csv.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 INGETIVE (<https://ingetive.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import contextlib
+import csv
+import base64
+import logging
+import cStringIO
+import os
+
+from odoo import models, fields, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from dbfpy import dbf
+except ImportError as err:
+    _logger.info(err)
+
+
+class Conversion(models.TransientModel):
+    _name = 'dbftocsv.conversion'
+    _description = 'File DBF to convert to CSV'
+
+    data_file = fields.Binary(
```

Comment:
Following your own convention, I'd call this `data_dbf`

Reply:
Ok